src/services.json

```diff
@@ -2,10 +2,12 @@
   "core": "/api/core",
   "mailer": "/api/mailer",
   "events": "/api/events",
+  "summeruniversity": "/api/summeruniversity",
```

Comment:
can we go with something shorter? like /api/su?

Reply:
It felt too short for me, but summeruniversity is also a bit too long. We can change it later as well